docs/topics/media-pipeline.rst

```diff
@@ -204,6 +206,18 @@ For information about authentication, see this `documentation`_.
 
 .. _documentation: https://cloud.google.com/docs/authentication/production
 
+You can modify the Access Control List (ACL) policy used for the stored files,
+which is defined by the :setting:`FILES_STORE_GCS_ACL` and
+:setting:`IMAGES_STORE_GCS_ACL` settings. By default, the ACL is set to
+``projectPrivate``. To make the files publicly available use the ``publicRead``
```

Comment:
From https://cloud.google.com/storage/docs/access-control/lists#predefined-acl:

> This is the default ACL for newly created buckets. This is also the default ACL for newly created objects unless the default object ACL for that bucket has been changed. 

Does it mean this PR is not backwards compatible for GCS buckets where default object ACL is changed - projectPrivate will be used, instead of a configured default? If so, do you think it is possible to add a special value (empty?) which tells "use whatever configured for a bucket", and have it by default?